test/rules/assertions/requireDescription.js

```diff
@@ -597,6 +597,25 @@ export default {
       ],
       parser: require.resolve('@typescript-eslint/parser'),
     },
+    {
+      code: `
+          interface quux {
+            /**
+             * If the thing should be checked.
+             */
+            checked?: boolean
+          }
+      `,
+      options: [
+        {
+          contexts: [
+            'TSPropertySignature',
+          ],
+          descriptionStyle: 'tag',
```

Comment:
Please remove this `descriptionStyle` line as it is otherwise reporting an error because there is no `@description` tag.